cherryontech/jupiter-jumpers
# Register form2

## Description

_1) What does this PR do? 
Registers users and set up Firebase 

2) List out what you did._
Configured Firebase into our code , set up sign up page with some validations.  Once user is signed up, will go straight to login.   

## Related Issue

[_Provide a link to the related issue in Jira._](https://cherryontech-jupiter-jumpers.atlassian.net/jira/software/projects/SCRUM/boards/1/timeline?selectedIssue=SCRUM-10)

## Related Story Card

_Provide a link to the relevant story card._

## Type of Changes

_Feature update/change/addition, bug fix, styling update, etc._

## Acceptance Criteria

_Include the checked-off acceptance criteria from the issue._

## Update Screenshots

_Provide screenshots of the view before and after the changes._

### After 
<img width="1455" alt="Screenshot 2024-04-23 at 4 15 25 PM" src="https://github.com/cherryontech/jupiter-jumpers/assets/123599985/50fd1b26-96d3-44ba-a681-d101710e3f53">


## Testing Instructions
-Fill out Register Form 
- Go to Firestore page and make sure user has been added to database.  
- Making sure Validations work according to instructions
- After registration, make sure you go to login page 
_In a list format, tell us how to test this PR._

## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._
